mssql-ballerina/Types.bal

```diff
@@ -0,0 +1,43 @@
+// // Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+// //
+// // WSO2 Inc. licenses this file to you under the Apache License,
+// // Version 2.0 (the "License"); you may not use this file except
+// // in compliance with the License.
+// // You may obtain a copy of the License at
+// //
+// // http://www.apache.org/licenses/LICENSE-2.0
+// //
+// // Unless required by applicable law or agreed to in writing,
+// // software distributed under the License is distributed on an
+// // "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// // KIND, either express or implied.  See the License for the
+// // specific language governing permissions and limitations
+// // under the License.
+
+// import ballerina/time;
```

Comment:
As discussed in the review, do not have commented out code in the PR

Reply:
Fixed with [`14793d0`](https://github.com/ballerina-platform/module-ballerinax-mssql/pull/2/commits/14793d087b1567bbd5636acb4dbff3097a38a554)